include/examples/simple_behavior.h

```diff
@@ -0,0 +1,192 @@
+  void Update() override
+  {
+    // Update joint velocities from robot
+    joint_velocities_ = robot_->GetJointVelocities();
+
+    switch (state_)
+    {
+      case RUNNING:  // in a running state
+        // Set joint torques to running levels
+        desired_torques_ =
+            std::vector<float>(robot_->joints.size(), RUN_TORQUE);
+        break;
+
+      case STOPPING:  // in a stopping state
+        // Lower the joint torques to slowing levels
+        desired_torques_ =
+            std::vector<float>(robot_->joints.size(), SLOW_TORQUE);
+
+        // Set behavior to inactive if average velocity is below a threshold
+        float avg_velocity =
+            std::accumulate(joint_velocities_.begin(),
+                            joint_velocities_.end(),
+                            0.0) / static_cast<float>(joint_velocities_.size());
+        if (avg_velocity <= 0.1)
+        {
+          set_inactive();
+          std::fprintf(stdout, "%s is no longer active\n", get_name().c_str());
+        }
+        break;
+    }
+
+    // Set joint torques
+    robot_->SetTorques(desired_torques_);
+  }
+
+  /**
+   * @brief Stops the behavior
+   * @details This code is run when a behavior is directed to stop running by
+   *          the behavior manager. This should trigger a shutdown process for
+   *          the behavior, e.g. slowing the motor speed. The upcoming behavior
+   *          is passed as an argument to permit different shutdown procedures
+   *          for different on-deck behaviors. In this example, the internal
+   *          state is modified to indicate the behavior should stop.
+   * @param next_behavior the upcoming behavior
+   */
+  void Stop(std::unique_ptr<kodlab::Behavior<SimpleRobot>> &next_behavior) override
+  {
+    // Indicate that behavior should stop
+    state_ = STOPPING;
+    std::fprintf(stdout, "Stopping %s\n", get_name().c_str());
+  }
+
+  /**
+   * @brief Returns the active state of the behavior
+   * @details This function identifies if the behavior is active. By default, it
+   *          simply accesses the `active_` variable. More advanced
+   *          implementations may use robot state to make this determination.
+   * @return true if behavior is running, false otherwise
+   */
+  bool Running() override { return active_; }
+
+  /**
+   * @brief Indicates if behavior is finished and ready to transition
+   * @details This function indicates to the behavior manager whether the
+   *          behavior is prepared to terminate and transition to the next
+   *          behavior. In its simplest form, this function simply accesses
+   *          the internal behavior active flag. This function can be used as a
+   *          mode switch guard.
+   * @param next_behavior the upcoming behavior
+   * @return true if behavior is ready to transition, false otherwise
+   */
+  bool ReadyToSwitch(std::unique_ptr<kodlab::Behavior<SimpleRobot>> &next_behavior) override
+  {
+    return !Running();
```

Comment:
Add the check on avg velocity here.

Reply:
Changed in 3c89ff3de8b8d2cd4de29d27e34cc7a88454cb86.